Editor/CreateBlendTree.cs

```diff
@@ -0,0 +1,32 @@
+using System.IO;
+using UnityEditor;
+using UnityEditor.Animations;
+using UnityEditor.ProjectWindowCallback;
+using UnityEngine;
+
+namespace nadena.dev.modular_avatar.core.editor
+{
+    public class CreateBlendTree : Editor
+    {
+        [MenuItem("Assets/Create/BlendTree", priority = 411)]
+        static void CreateNewBlendTree()
+        {
+            ProjectWindowUtil.StartNameEditingIfProjectWindowExists(
```

Comment:
Is this an undocumented API?

Reply:
It's seemingly an older API. Seems to be missing from later unity version documentation. There's a page on end rename action, but not start.
https://docs.unity3d.com/ScriptReference/ProjectWindowCallback.EndNameEditAction.Action.html

But it is in use by default assets
https://github.com/Unity-Technologies/UnityCsReference/blob/496f7d6c5c0882f35bb776e96356712d54710033/Editor/Mono/ProjectBrowser/ProjectWindowUtil.cs#L347

What this ends up doing is beginning to rename a new asset, confirming the name then actually creating it, like any default asset you create.